### PR TITLE
fetch url and display cards on page

### PR DESCRIPTION
**Pull request by Sanae-G**
fetched url and displayed cards on page

### CODE REVIEW

File: src/components/CardList.jsx [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+  const [characters, setCharacters] = useState([]);
+
+  const Characters =() =>{
+    fetch(apiURL).then((response) => response.json()).then((data) => setCharacters(data.results))
+  }
+
+
+  useEffect(() => {
+    Characters();
+  }, []);
```
Comment (AlexF090) [inline src/components/CardList.jsx]: Thomas gave the tip today to write the `fetch()` function in the `App.jsx`. Will be necessary later.
So I would recommend you to do that already now.

File: src/components/CardList.jsx [hunk 2]
```diff
@@ -1,47 +1,47 @@
 import styled from 'styled-components';
+import { useEffect, useState } from "react";
 
 function CardList() {
+
+  const apiURL="https://rickandmortyapi.com/api/character";
+  const [characters, setCharacters] = useState([]);
+
+  const Characters =() =>{
```
Comment (ThomasF85) [inline src/components/CardList.jsx]: this is a function and should therefore be written in non capital - camel case letters. The name should also represent what the function is doing, e.g. fetchCharacters

File: src/components/CardList.jsx [hunk 3]
```diff
@@ -1,47 +1,47 @@
...
+     );
+   })}
+  </CardListStyled>
+  )}
+
 
 export default CardList;
 
-const CardContent = styled.div`
+const CardContent = styled.li`
```
Comment (AlexF090) [inline src/components/CardList.jsx:36]: great

File: src/components/CardList.jsx [hunk 4]
```diff
@@ -1,47 +1,47 @@
...
     border: 1px solid black;
     border-radius: 0.5em;
     height: 10%;
     display: flex;
     flex-direction: column;
 `;
 
-const CardListStyled = styled.section`
+const CardListStyled = styled.ul`
```
Comment (AlexF090) [inline src/components/CardList.jsx:44]: good idea ;)

File: src/App.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ import CardList from './components/CardList';
 import Navbar from './components/Navbar';
 
 function App() {
```
Comment (ThomasF85) [inline src/App.js:6]: Nice structure!

**Review by ThomasF85 — APPROVED** [review]
Good job! There is one renaming that should be done.

File: src/components/CardList.jsx [hunk 6]
```diff
@@ -1,47 +1,47 @@
...
+       <CardContent key={id}>
+          <img src={image}>
+          </img>
+         <CharacterName>{name}</CharacterName>
+         <button>Show more</button>
+       </CardContent>
+     );
+   })}
+  </CardListStyled>
+  )}
```
Comment (AlexF090) [inline src/components/CardList.jsx:31]: 👍🏻

**Review by AlexF090 — CHANGES_REQUESTED** [review]

File: src/components/CardList.jsx (hunk 1 above)
Comment (Sanae-G) [reply src/components/CardList.jsx]: Thank you very much Alex. I will do it in my next step. 👍

**Pull request merged**